plugin.xml

```diff
@@ -119,7 +119,7 @@
 
 		<source-file src="src/android/src/com/pushwoosh/plugin/pushnotifications/NotificationFactory.java"
 						target-dir="src/com/pushwoosh/plugin/pushnotifications" />
-
+        <framework src="com.google.android.gms:play-services-ads:15.+" />
```

Comment:
fix the indentation please :)

Reply:
Why do you need ads framework? The official plugin doesn't use it.